src/transactions/AllowTrustOpFrame.cpp

```diff
@@ -59,7 +55,8 @@ bool
 AllowTrustOpFrame::doApply(AbstractLedgerTxn& ltx)
 {
     ZoneNamedN(applyZone, "AllowTrustOp apply", true);
-    if (ltx.loadHeader().current().ledgerVersion > 2)
+    auto ledgerVersion = ltx.loadHeader().current().ledgerVersion;
```

Comment:
That's an interesting idea that I hadn't considered. The point about the exact behavior is a good one. I didn't think `AllowTrustOpFrame` would need to change in the future though since it should never be able to modify any future flags, so there shouldn't be anything to copy/paste. I think If we find ourselves in that position in the future, we can implement your idea then.

Reply:
yes that's fine - I am thinking more about some of the edge cases that come with auth revocation